.coveragerc

```diff
@@ -0,0 +1,5 @@
+[report]
+exclude_lines =
+    raise NotImplementedError
+    raise RuntimeError
+    except KeyError
```

Comment:
If this is here for the sole purpose of reaching 100% code coverage, we can drop it.
I'm ok with our current 98%. :-)

Reply:
Why drop it?
Those exclusions make sense and improve coverage and also are sensitive to decreased coverage over-time (As did happen in [#710)](https://github.com/TomerFi/aioswitcher/pull/710)

Reply:
As you wish.. File removed.

Reply:
Thank you very much for your hard work - merging this now.
For future reference, this repository is already full of annoying configuration files, whenever possible, it's prefered using `pyproject.toml'. Coverage is already configured there:

https://github.com/TomerFi/aioswitcher/blob/0544a5047ae1e9ae738f5b909b67c9af793e355e/pyproject.toml#L100